tommath_private.h

```diff
@@ -236,6 +236,13 @@ MP_DEPRECATED(s_mp_toom_mul) mp_err mp_toom_mul(const mp_int *a, const mp_int *b
 MP_DEPRECATED(s_mp_toom_sqr) mp_err mp_toom_sqr(const mp_int *a, mp_int *b);
 MP_DEPRECATED(s_mp_reverse) void bn_reverse(unsigned char *s, int len);
 
+#define MP_GET_ENDIANNESS(x) \
+   do{\
+      int16_t n = 0x1;                                          \
```

Comment:
I already see the "shadowing variable name" warnings pop up ...

Reply:
Yes, I would really like if we could use inline functions. But given the limited use of the macro, I think it is ok. There are also no good alternatives. @czurnieden tried writing it first as expression but then there is a c++ incompatibility.